rever/environ.xsh

```diff
@@ -160,6 +160,8 @@ ENVVARS = {
         'the container itself, default $HOME/$PROJECT'),
     'GITHUB_CREDFILE': ('', is_string, str, ensure_string,
                         'GitHub credential file to use'),
+    'GITHUB_FORK_ORG': ('', is_string, str, ensure_string, 
```

Comment:
Should this be a global env var or something specific to the CondaForge activity?